### PR TITLE
feat: error response for not id found on updateUser

### PR DESCRIPTION
**Pull request by AdrianValdes**

### CODE REVIEW

File: src/controller/userController.js [hunk 1]
```diff
@@ -28,10 +28,16 @@ const addUser = async (req, res, next) => {
...
+    const updatedUser = await User.findByIdAndUpdate(
+      id,
+      req.body,
+      {
+        new: true,
+        useFindAndModify: false,
+      },
+      (err, model) => next(err)
+    );
+
```
Comment (chmette) [inline src/controller/userController.js:46]: tests in insomnia are showing that i cant update a user after these change , server error 404 - cannot find requested ressource.

Comment (AdrianValdes) [reply src/controller/userController.js:46]: True,  I am on it, try again please.

File: src/controller/errorController.js [hunk 2]
```diff
@@ -5,13 +5,16 @@ const {
...
   try {
+    if (err.status === 401) {
+      res.status(err.status).send({
+        messages: err.message,
+      });
+    }
+    if (err?.kind === 'ObjectId') {
+      res.status(422).send({ messages: err.reason.message });
+    }
+
```
Comment (chmette) [inline src/controller/errorController.js:17]: What does err.kind do?

File: src/controller/userController.js (hunk 1 above)
Comment (chmette) [reply src/controller/userController.js:46]: nope still not

Comment (chmette) [reply src/controller/userController.js:46]: now it works

File: .eslintrc.json [hunk 3]
```diff
@@ -16,14 +16,17 @@
...
+  ],
   "rules": {
     "import/prefer-default-export": "off",
     "import/no-default-export": "error",
     "mongodb/no-replace": 0,
     "no-unused-vars": [
       "error",
       {
-        "argsIgnorePattern": "next"
+        "args": "none"
```
Comment (chmette) [inline .eslintrc.json:29]: could you shortly explain these please?

**Review by chmette — APPROVED** [review]
please answer my questions that i can understand what it does

File: src/controller/errorController.js (hunk 2 above)
Comment (AdrianValdes) [reply src/controller/errorController.js:17]: You can check the type or error:
1. https://mongoosejs.com/docs/api.html#error_Error-ValidatorError
2. https://mongoosejs.com/docs/validation.html#validation-errors

File: .eslintrc.json (hunk 3 above)
Comment (AdrianValdes) [reply .eslintrc.json:29]: It is to change how the eslint rule works with arguments, did you try to find it? [They](https://eslint.org/docs/rules/no-unused-vars#args) explain it better than me

File: src/controller/errorController.js (hunk 2 above)
Comment (chmette) [reply src/controller/errorController.js:17]: you are an error profi now :)

**Pull request merged**